_posts/2017-03-22-hololens.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: 'Mapping met de HoloLens'
+category: hololens
+tags: hololens mapping spatialmapping mixed reality 
+comments: true
+author: ruben
+min_read: 7
+---
+
+Hoe de toekomst van computergebruiker eruit gaat zien? Microsoft denkt daar een antwoord op te hebben met de HoloLens. De bril die de echte wereld mengt met virtuele beelden. Microsoft heeft hoge verwachtingen van de HoloLens, die vooralsnog alleen beschikbaar is voor ontwikkelaars. 
+
+Het verschil met andere wearables zoals de Google Glass van enkele jaren geleden is dat deze alleen een platte laag toevoegen. De HoloLens gaat verder en koppelt virtuele objecten aan bepaalde plekken die gebruikers van alle kanten kunnen zien. Microsoft noemt deze virtuele objecten hologrammen. 
+
+![HoloLens](/assets/hololens.jpg)
+
+Om hologrammen te kunnen plaatsen zonder dat deze in de muur, tafel of de vloer verdwijnt moet de ruimte gescand worden. Dit gebeurt op basis van spatial mapping. Voor deze techniek herbergt de HoloLens dieptesensoren, een rgb-camera en infrarood-projectoren om een blikveld van 120 bij 120 graden in kaart te brengen. De positie van het hoofd van de gebruiker wordt berekend met behulp van een accelerometer, gyroscoop en megnetometer.
```

Comment:
zonder dat deze in de muur, tafel of vloer **verdwijnen**, moet de ruimte gescand worden